smart_proxy_discovery.gemspec

```diff
@@ -18,5 +18,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency('mocha')
   s.add_development_dependency('webmock')
 
+  s.add_runtime_dependency('mime-types', ['= 1.25.1'])
```

Comment:
Yeah, rest-client 1.7 dropped 1.8 support too, so they probably both have to be pinned.